data/zonetool/localizedstrings/russian.json

```diff
@@ -0,0 +1,598 @@
+    "LUA_MENU_PRESTIGE": "Престиж",
+    "LUA_MENU_PRESTIGE_DESC": "Изменение уровня престижа.",
+    "LUA_MENU_RANK": "Ранг",
+    "LUA_MENU_RANK_DESC": "Изменение ранга.",
+    "LUA_MENU_UNSAVED_CHANGES": "Изменения не были сохранены, вы уверены, что хотите выйти?",
+    "LUA_MENU_SAVE": "Сохранить изменения",
+    "LUA_MENU_SAVE_DESC": "Внесение изменений в игру.",
+    "LUA_MENU_SETTINGS": "Параметры",
+    "LUA_MENU_EDIT_STATS": "Корректировка статистики",
+
+    "UPDATER_POPUP_NO_UPDATES_AVAILABLE": "У вас установлены все последние обновления",
+    "UPDATER_POPUP_AVAILABLE_UPDATE_TEXT": "Доступно обновление клиента игры,\nначать установку сейчас?",
+    "UPDATER_POPUP_SUCCESSFUL": "Обновление завершено",
+    "UPDATER_POPUP_RESTART_POPUP_TEXT": "Для применения изменений необходим перезапуск игры",
+    "UPDATER_POPUP_CHECKING_FOR_UPDATES": "Проверка наличия обновлений...",
+
+    "MPHUD_FPS": "К/С: ",
+    "MPHUD_LATENCY": "Задержка: ",
+    "MPHUD_LATENCY_MS": " мс",
+    "LUA_MENU_TELEMETRY": "ТЕЛЕМЕТРИЯ",
+
+    "LOCALE_0": "Английский",
+    "LOCALE_1": "Французский",
+    "LOCALE_2": "Немецкий",
+    "LOCALE_3": "Итальянский",
+    "LOCALE_4": "Испанский",
+    "LOCALE_5": "Русский",
+    "LOCALE_6": "Польский",
+    "LOCALE_7": "Португальский",
+    "LOCALE_8": "Японский",
+    "LOCALE_9": "Японский (английская озвучка)",
+    "LOCALE_10": "Китайский традиционный",
+    "LOCALE_11": "Китайский упрощенный",
+    "LOCALE_12": "Арабский",
+    "LOCALE_13": "Чешский",
+    "LOCALE_14": "Испанский (Латинская Америка)",
+    "LOCALE_15": "Корейский",
+    "LOCALE_16": "Английский цензурный",
+
+    "LUA_MENU_MULTIPLAYER_CAPS": "H1-MOD: СЕТЕВАЯ ИГРА\n",
+    "MENU_MULTIPLAYER_CAPS": "H1-MOD: СЕТЕВАЯ ИГРА",
+
+    "PLATFORM_SYSTEM_LINK_TITLE": "СПИСОК СЕРВЕРОВ",
+
+    "MPUI_MP44": "MP-44",
+    "WEAPON_AT4": "AT4",
+    "WEAPON_BARRETT": "Barrett .50",
+    "WEAPON_BERETTA": "M9",
+    "WEAPON_COLT1911": "M1911 .45",
+    "WEAPON_COLT45": "M1911 .45",
+    "WEAPON_DESERTEAGLE55": "Командирский Дезерт Игл",
+    "WEAPON_DESERTEAGLEGOLD": "Золотой Дезерт Игл",
+    "WEAPON_HK79": "HK79",
+    "WEAPON_MELEESHOVEL": "Могильщик",
+    "WEAPON_MP44": "MP-44",
+    "WEAPON_MP5_SILENCER": "MP5 с глушителем",
+    "WEAPON_NO_AMMO_CAPS": "НЕТ ПАТРОНОВ",
+    "WEAPON_NO_FRAG_GRENADE": "Осколочных гранат не осталось",
+    "WEAPON_NO_SPECIAL_GRENADE": "Особых гранат не осталось",
+    "WEAPON_P90_SILENCER": "P90 с глушителем",
+    "WEAPON_SILENCER_ATTACHMENT": "с глушителем",
+    "WEAPON_USP": "USP .45",
+
+    "EXE_SAY": "^3Всем^7",
+    "EXE_SAYTEAM": "^5Команде^7",
+
+    "MENU_SB_TOOLTIP_BTN_REFRESH": "Обновить список",
+    "MENU_TYPE1": "Режим",
+    "SERVERLIST_ADD_TO_BLACKLIST": "Добавить в ЧС",
+    "SERVERLIST_REMOVE_FROM_BLACKLIST": "Убрать из ЧС",
+    "LUI_MENU_BLACKLIST": "Черный список",
+
+    "MPUI_ATDM_RECIPE_NAME": "Usilenie",
```

Comment:
Translit as workaround for INVALIDCHARS error caused by Cyrillic in recipe auto-generated name